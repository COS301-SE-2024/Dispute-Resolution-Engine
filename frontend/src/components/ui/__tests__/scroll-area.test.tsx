import renderer from 'react-test-renderer';
import { ScrollArea } from '../scroll-area';

it('renders correctly', () => {
  const tree = renderer
    .create(<ScrollArea />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
