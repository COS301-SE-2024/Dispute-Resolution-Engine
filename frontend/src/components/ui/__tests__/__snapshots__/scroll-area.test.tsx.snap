// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="relative overflow-hidden"
  dir="ltr"
  style={
    {
      "--radix-scroll-area-corner-height": "0px",
      "--radix-scroll-area-corner-width": "0px",
      "position": "relative",
    }
  }
>
  <style
    dangerouslySetInnerHTML={
      {
        "__html": "[data-radix-scroll-area-viewport]{scrollbar-width:none;-ms-overflow-style:none;-webkit-overflow-scrolling:touch;}[data-radix-scroll-area-viewport]::-webkit-scrollbar{display:none}",
      }
    }
  />
  <div
    className="h-full w-full rounded-[inherit]"
    data-radix-scroll-area-viewport=""
    style={
      {
        "overflowX": "hidden",
        "overflowY": "scroll",
      }
    }
  >
    <div
      style={
        {
          "display": "table",
          "minWidth": "100%",
        }
      }
    />
  </div>
</div>
`;
